3p10.c

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int MinimumNoOfBank (int amount, int banknotes[6])
+{
+    int i = 5, minNumber = 0;

```

Comment:
This `i` could it be initialized from a global constant? for example
```
int i = NUMBER_OF_BANKNOTES;
```

and above all the functions, you could have

```
const int NUMBER_OF_BANKNOTES = 5;
```